src/main/java/red/storage/Storage.java

```diff
@@ -0,0 +1,136 @@
+package red.storage;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import java.util.Scanner;
+
+import red.command.AddCommand;
+import red.command.Command;
+
+import red.task.TaskList;
+import red.task.DeadlineTask;
+import red.task.EventTask;
+import red.task.ToDoTask;
+
+import red.ui.UI;
+
```

Comment:
I like how the imports are separated with a spacing into each category, very organised and tidy to read